modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/keepProductInCheckoutFromRemovalModal.js

```diff
@@ -0,0 +1 @@
+export default () => cy.get('[data-cy=remove-modal-keep-button]').click()
```

Comment:
This page object seems to be unused

Reply:
yup, created for the next PR already. Altogether with the `data-cy` attribute in the source file. I decided to keep it here, even though it is not used yet.